ios/src/TGLabelPickResult.h

```diff
@@ -0,0 +1,24 @@
+//
+//  TGLabelPickResult.h
+//  tangram
+//
+//  Created by Karim Naaji on 11/02/16.
+//
+
+#import <Foundation/Foundation.h>
+#import "TGGeoPoint.h"
+
+typedef NS_ENUM(NSInteger, TGLabelType) {
+    TGLabelTypeIcon = 0,
+    TGLabelTypeText,
+};
+
+@interface TGLabelPickResult : NSObject
+
+- (id) initWithCoordinates:(TGGeoPoint)coordinates type:(TGLabelType)type properties:(NSDictionary *)properties;
+
+@property (readonly, nonatomic) TGGeoPoint coordinates;
```

Comment:
Why the `coordinates` should be `readonly` and `assign` at the same time?
Updating for 
```
+@property (readonly, copy, nonatomic) TGGeoPoint coordinates;
```